ide-backend-common/IdeSession/GHC/API.hs

```diff
@@ -16,8 +16,8 @@ module IdeSession.GHC.API (
   , sourceExtensions
   , cabalMacrosLocation
     -- * Paths
-  , ideSessionSourceDir
-  , ideSessionDataDir
+--  , ideSessionSourceDir
```

Comment:
I think we should remove code rather than commenting it out.  That's what source control is for, after all!